marketplacetests/marketplace/pages/base.py

```diff
@@ -137,6 +129,18 @@ def show_menu(self):
             lambda m: 'mkt-nav--visible' in body.get_attribute('class'))
         return NavMenu(self.marionette)
 
+    def search_for_paid_apps(self):
+        self.set_region('us')
+        return self.search(':paid')
+
+    def is_app_installed(self, name):
```

Comment:
It uses `switch_to_marketplace_frame` which exists in the app object, so by default it will not have access to that if we put it into `MarketplaceGaiaTestCase`. For that reason I would prefer to leave it as is.